test/bdd/features/didexchange_public_did.feature

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Reference : https://github.com/hyperledger/aries-rfcs/tree/master/features/0023-did-exchange
+
+@all
+Feature: Decentralized Identifier(DID) exchange between the agents using public did in invitation
+
+  @didexchange_public_did
+  Scenario: did exchange e2e flow using public DID in invitation
+    Given "Maria" agent is running on "localhost" port "random" with http-binding did resolver url "http://localhost:48326/.sidetree/document" which accepts did method "sidetree"
```

Comment:
How do we know the port is `48326`?

Reply:
e.g., can this be an environment variable so it matches whatever the docker config is?

Reply:
Created new issue #532